Lib/test/cppextdata/setup.py

```diff
@@ -1,5 +1,6 @@
 # gh-91321: Build a basic C++ test extension to check that the Python C API is
 # compatible with C++ and does not emit C++ compiler warnings.
+import os
 import sys
 from test import support
 
```

Comment:
Since this moved to a new directory, that directory needs to be added to `TESTSUBDIRS` in `Makefile.pre.in` to be installed.

Reply:
Done! Thanks for catching this!